src/nfa.rs

```diff
@@ -22,7 +22,7 @@ pub struct Transition {
 
 #[derive(Debug, Clone)]
 pub struct Nfa {
-    states: Vec<String>,
+    pub states: Vec<String>,  // made public for CSV outputs
```

Comment:
Can be left private in case we add a getter method, may be more appropriate dive we never wore to this